ethereum/contracts/ETHVault.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+// Holds ETH on behalf of ETHApp
+contract ETHVault is Ownable {
```

Comment:
I'd like us to use `Ether` instead of `ETH` in names. Ether is the proper name of the asset and it reads a bit better than the all-caps ETH. I've been wanting to update our contracts/pallets but its very low-hanging fruit.

Can just rename this contract and filename for now.